scripts/pnpm-install.sh

```diff
@@ -9,10 +9,10 @@ fi
 # After installing everything the first time, reread the graph and see if anything was missing..
 # If something was missing, we install that again and repeat until nothing is missing.
 
-echo pnpm install --filter . --filter ...[HEAD^1]...
-pnpm install --filter . --filter ...[HEAD^1]...
+echo pnpm install --filter . --filter ...[origin/master]...
```

Comment:
For PRs, this is identical. HEAD^1 is the merge base which is master. For commits to master, this is the previous commit which is also desirable.